consensus/consensus.md

```diff
@@ -134,8 +134,8 @@ nonce:
 #### Key Blocks
 
 ```
-PROTOCOL_VERSION: 21
-GENESIS_VERSION: 21
+PROTOCOL_VERSION: 22
+GENESIS_VERSION: 22
 ```
 
 - the timestamp of a key block MUST be smaller than `now() + 10m`
```

Comment:
I did not find any requirements on adapting the 11 timestamp median-related number of blocks at the line below with the 17 of the target adjustment. So this does not seem requiring change.